yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/docs/request/task/SandboxTextExtractionTaskRecommendation.java

```diff
@@ -0,0 +1,75 @@
+package com.yoti.api.client.sandbox.docs.request.task;
+
+import com.yoti.api.client.sandbox.docs.SandboxDocScanConstants;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SandboxTextExtractionTaskRecommendation {
+
+    @JsonProperty("value")
+    private final String value;
+
+    @JsonProperty("reason")
+    private final SandboxTextExtractionTaskReason reason;
```

Comment:
Does the `reason` property need to be excluded when recommendation value is `PROGRESS`?